server.js

```diff
@@ -31,11 +35,88 @@ app.get('/api', function(req, res) {
 
   countryTrends.returnTrendsByCountry(function(results) {
     console.log("results", results);
-    res.status(201);
+    res.status(200);
     res.send(results);
   });
 });
 
+app.get('/db', function(req, res) {
+  Trend.find(function(err, trends) {
+    if (err) {
+      throw err;
+    }
+    console.log('TRENDS', trends);
```

Comment:
Ah yes, thank you.  This is expected.  Hopefully we'll be able to populate a shared database and pull from that in the future.